trento/xml/article_sap_trento.xml

```diff
@@ -1678,618 +1677,48 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
-            (SID), a summary of the database instances and their status and custom
-            tags that you can manuall set/remove.
-            By clicking on the row of one particular database, a list of all related
-            instances is displayed and for each one of them you can see the corresponding
-            health status, its instance number, the feature (role) it provides, the system
-            replication role and status (when applicable), the cluster name (when applicable)
-            and the host where it is running.
-          </para>
-        </listitem>
-        <listitem>
-          <para> Ability to filters &hana; databases by health status, SID, and custom tags.
-          </para>
-        </listitem>
-      </itemizedlist>
-      <para> Clicking on one of the <guimenu>SID</guimenu>s opens the
-        <guimenu>HANA Databases</guimenu> detail view which shows
-        the following: </para>
-      <itemizedlist>
-        <title><guimenu>HANA Database</guimenu> details view</title>
-        <listitem>
-          <para>
-            <guimenu>Layout</guimenu> section: lists all related
-            &hana; instances with their corresponding virtual hostname, instance number,
-            features (roles), HTTP/HTTPS ports, start priorities, and SAPControl status.
-          </para>
-        </listitem>
-        <listitem>
-          <para>
-            <guimenu>Hosts</guimenu> section: lists the hosts where all the related
-            instances are running and for each host it provides the host name, the
-            local IP address(es), the cloud provider (when applicable), the cluster
-            name (when applicable), the system ID, and the &t.agent; version.
-            </para>
-          <para> Clicking on a host name takes you to the
-            corresponding <xref linkend="lst-trento-web-hosts-details" />.
-          </para>
-        </listitem>
-      </itemizedlist>
-      <figure>
-        <title>HANA Database details</title>
-        <mediaobject>
-          <imageobject>
-            <imagedata fileref="trento-web-hana-database.png" width="70%"/>
-          </imageobject>
-        </mediaobject>
-      </figure>
-    </section>
-
-    <section xml:id="sec-trento-configuration-checks">
-      <title>HA configuration checks</title>
-      <para> To get an overview of the configuration checks that Trento
-        can manage click the <guimenu>Checks catalog</guimenu>
-        entry in the left sidebar.
-      </para>
-      <figure>
-        <title><guimenu>Checks catalog</guimenu></title>
+      </figure>  
+   <para>Once the SUSE Manager settings are entered the SAP admin can test the 
+   connection by clicking on the Test Connection button. If the connection is successfull,
+   from that moment onward, for any registered host that is also managed by the SUSE Manager
+   instance which connection details we have provided, the corresponding Host Details view will
```

Comment:
The sentence starting with "If the connection is successful..." is quite long and hard to follow. Can we split it up into smaller chunks? Shouldn't the "is" be replaced with a "was"?

Reply:
My English is not that good. I defer the decision to an English native speaker.

Reply:
What about this:

```xml
With the &suma; settings configured, the &sap; admin can test the
connection by clicking <guimenu>Test Connection</guimenu>.

If the connection is successful, the Host Details view displays
a summary of the available patches and the upgradable packages
for each host managed by SUMA
```